ui/app/services/permissions.js

```diff
@@ -0,0 +1,123 @@
+import Service, { inject as service } from '@ember/service';
```

Comment:
Could we add some comments about what this service is for, etc. ? (I know I've been bad about leaving comments, but trying to get better at it!)

Reply:
👍